CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ option(ENABLE_SANITIZE "Sanitize tool" OFF)
 option(ENABLE_SECURITY "Security Ford protocol protection" ON)
 option(ENABLE_HMI_PTU_DECRYPTION "Policy table update parsed by hmi" ON)
 option(ENABLE_EXTENDED_POLICY "Turns extended flow which requires embedded system interaction" ON)
+option(ENABLE_THREAD_COUNT "Turns thread count to log file" OFF)
```

Comment:
@OHerasym it's not clear for me what does it mean `Turns thread count to log file`